frontend-build-pipeline.yml

```diff
@@ -16,27 +19,48 @@ parameters:
 - name: sonarCloudProjectName # sonar cloud's cli project name
   type: string
   default: 'RevatureShopFrontend'
+- name: sonarCloudExclude # files for sonar cloud to ignore
+  type: string
+  default: '**/*.spec.ts'
+- name: sonarCloudCoverageDir
+  type: string
+  default: '$(Build.Repository.LocalPath)/coverage/lcov.info'
+- name: sonarCloudSources
+  type: string
+  default: '.'
 
 steps:
 
-
-
 # Node.js is required to build Angular apps
 - task: NodeTool@0
   inputs:
     versionSpec: '12.14'
   displayName: 'Install Node.js'
 
 - script: |
-    npm ci
+    ls $(Build.Repository.LocalPath)
```

Comment:
looks like leftover debug stuff

Reply:
My bad, that shouldn't have been left in there. Cleaned up.